### DISCUSSION TITLE
Unit tests / mocks

### DISCUSSION THREAD

**Discussion by PaulRudin** [honojs/hono#3348] [General]
I'm using Hono for a Fastly compute service.

The problem I have is that I'm using some things that (I assume) are only provided by the Fastly runtime (e.g. import 'fastly:config-store').

But if I try to use node then I get errors since these things don't exist.

Are there any examples for this?

**Comment by thenbe** [comment]
I often find that this vitest [mocking cheatsheet](https://vitest.dev/guide/mocking.html#cheat-sheet) is a great place to start for stuff like this.
